### ISSUE TITLE
As a user, I need to know that permits are not transferable so that I don't inadvertently break the rules

### ISSUE THREAD

**Issue by MelissaBraxton** [nciinc/fs-intake-module#211]
## Acceptance Criteria
- [ ]

## Definition of Done
- [ ] Code unit / e2e tested and comply with QWASP
- [ ] No new security vulnerabilities
- [ ] Internal team code review
- [ ] Documentation / readme.md / CircleCI updated
- [ ] Docker updated
- [ ] Compare finished design with mockup
- [ ] Usability tested

**Comment by JaneZC** [comment]
@ASprinkle @MelissaBraxton 

I had a question/clarification need on this one and #218. I believe the current version of the permit already contains language mentioning the need for the permit holder to be present at the time trees are cut down. So, I think this one is more focused on making sure content is on the webpages.

I have been wondering whether/if permit-specific requirements needed to be added to the Rules to Know section; it would seem a logical place for it (and someone would need to check they have read, understood and agree to it). 

So, content for this story and 218 would perhaps fit better in a story that describes adding permit-specific 'rules' to the Rules to Know section. (So, other content would be things like putting the permit on the dashboard, permit must be completed before transporting tree, etc.)

**Comment by sadlerw** [comment]
I think adding to rules to know is a good idea. Is it also needed on confirmation page?  Please create a story for adding other things to the "Rules to know" section.

**Comment by bryanmcfadden** [comment]
The current language on the permit states: "Permit holder must be present at time of tree cutting."

**Comment by sadlerw** [comment]
New page screen shots:

![Screen Shot 2018-01-16 at 9.59.47 AM.png](https://images.zenhubusercontent.com/595e640361b4ca3b0a7370e7/d29a341a-5f93-492f-a688-b67fdcac1fd9)

![Screen Shot 2018-01-16 at 9.55.45 AM.png](https://images.zenhubusercontent.com/595e640361b4ca3b0a7370e7/0c28c40c-523e-4025-9d53-e9e7e9529853)

![Screen Shot 2018-01-16 at 10.09.18 AM.png](https://images.zenhubusercontent.com/595e640361b4ca3b0a7370e7/362f48a5-74d1-4737-9dc6-fe3bc898e6a7)

**Comment by MelissaBraxton** [comment]
A few minor tweaks:
- Will users know what a "printed permit" is? Consider changing to something like "You must print your permit in order for it to be valid..." Curious what @JaneZC thinks. 
 - Suggest grouping this info with like information under "Rules for your permit"

**Comment by sadlerw** [comment]
I think we added that as bold at the top so that users would not begin to fill out the form without realizing that they needed a printer.  We could add it to the permit rules section and also include the need for printing on white 8 1/2 x 11 too though.

**Comment by bryanmcfadden** [comment]
@MelissaBraxton @sadlerw @JaneZC - Will is correct. We were looking to inform users early about the need for printing the permit to be valid. (This will require more user testing as well to validate placement). How about this?

![image](https://user-images.githubusercontent.com/16689692/35012481-df780de8-fad7-11e7-886c-6f63ce33c49d.png)

**Comment by JaneZC** [comment]
@bryanmcfadden @MelissaBraxton I think that works.

In thinking about this some more, the only thing I think we need to be careful of is making an assumption that folks will print the permit as soon as they order it. We may want to add language to the email as well? 

I know that usability testing showed this was an issue for those printing immediately upon ordering the permit; not sure about those who  don't print right away. We can add that as a task on a future story, if everyone thinks it would be helpful. :)

**Comment by JaneZC** [comment]
@MelissaBraxton  @bryanmcfadden Sorry! I don't think I answered Melissa's question. I agree that the phrasing 'printed permit' is a bit awkward. (So much for attempting to cut down on wordiness!!)

I'd recommend:

Your permit must be printed to be valid and cannot be stored on a mobile device.

**Comment by sadlerw** [comment]
![Screen Shot 2018-01-16 at 5.22.04 PM.png](https://images.zenhubusercontent.com/595e640361b4ca3b0a7370e7/2414c310-29da-42e6-960a-6561aa63a693)

**Comment by sadlerw** [comment]
Should we put that in permit rules section as well?

**Comment by JaneZC** [comment]
@sadlerw Yes, I think so. 

I know we had talked about adding all the permit rules to that section; I got confused, though, because two different stories had text regarding permit rules being added to that section. 

Should I create a separate story for that (ie. seeing all permit rules in the rules section of the order confirmation page) or should it be a task on here?

**Comment by sadlerw** [comment]
@JaneZC a separate story please as I said above - this story is supposed to only be for permits not being transferrable.

**Comment by sadlerw** [comment]
OK this url: applications/christmas-trees/forests/arp/new

this is in the permit rules:

![Screen Shot 2018-01-17 at 8.41.23 AM.png](https://images.zenhubusercontent.com/595e640361b4ca3b0a7370e7/ccda0dae-3369-4532-9e26-2f552365c3b4)

this is the top:

![Screen Shot 2018-01-17 at 8.43.28 AM.png](https://images.zenhubusercontent.com/595e640361b4ca3b0a7370e7/da83068e-bf87-4d17-a240-5b17cff81b81)

**Comment by tram** [comment]
Sorry for jumping in late; have been out of the office in a National Forest.

The [version above that mentioned displaying it on the dash](https://github.com/nciinc/fs-permit-platform/issues/211#issuecomment-358107076) seems the clearest if the sentences are flipped and trimmed.

>  Your permit must be printed and clearly visible on your dashboard while tree cutting. Permits on mobile devices are not valid.

**Comment by JaneZC** [comment]
@tram I think we opted to remove the content related to the dashboard on this page; it is a bit premature to mention it here.

I think this is good to go?

**Comment by ASprinkle** [comment]
This is great! "Permit holder must be present at time of tree cutting"
Please add, "and transport". so that it looks like "Permit holder must be present at during tree cutting and transport".  Or if transport is weird, we can say "removal".

It's excellent, that the "Permit holder" language is consistent with what is in the name box of the permit.

**Comment by tram** [comment]
👍 LGTM. Ship it!
